dev-python/pykcs11/pykcs11-1.5.17.ebuild

```diff
@@ -1,32 +1,36 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
+DISTUTILS_EXT=1
 DISTUTILS_USE_PEP517=setuptools
 PYTHON_COMPAT=( python3_{11..13} )
 
 inherit distutils-r1
 
-MY_PN=PyKCS11
-MY_P=${MY_PN}-${PV}
+MY_P="PyKCS11-${PV}"
 
 DESCRIPTION="A complete PKCS#11 wrapper for Python"
 HOMEPAGE="https://github.com/LudovicRousseau/PyKCS11"
-SRC_URI="https://codeload.github.com/LudovicRousseau/${MY_PN}/tar.gz/${PV} -> ${P}.tar.gz"
+SRC_URI="https://github.com/LudovicRousseau/PyKCS11/archive/refs/tags/${PV}.tar.gz"
```

Comment:
Please add ``-> ${P}.gh.tar.gz``` here. This is the best practice according to Gentoo books

Reply:
Will do ! have you a link to it ? I've quickly search in devmanual and wiki but didn't find anything about it and I'd like to follow (all) the best practice

Reply:
https://pkgcore.github.io/pkgcheck/man/pkgcheck.html
```PythonGHDistfileSuffix
Distfile from GitHub is missing “.gh.tar.gz” suffix.```

just run ```pkgcheck scan```